packages/viewer/src/components/viewer-view-cube/viewer-view-cube.css

```diff
@@ -69,7 +69,7 @@
   position: unset;
   width: 100%;
   height: 100%;
-  overflow: visible;
+  overflow: visible !important;
```

Comment:
Is the `!important` required here? Usually a smell, but given the state of this prior, definitely open to this! :)

Reply:
BTW, tested locally with a linked version.... I'm so glad you found a fix for this!

Reply:
Yes unfortunately. The change to scoped styles changes the specificity of the selectors.